### PR TITLE
Web program manager

### PR DESCRIPTION
**Pull request by anthony-bartman**
# Date
Initial PR Date:
4/21/21
Due Date:
4/23/21

# Reviewers
@Ivendrakeo  @anthony-bartman @DeclanBruce @rockboynton @iliescua 

# Description
Merging final program manager functionality from backend into master. According to @DeclanBruce, all should be good to go for program manager.

# Project Issue
#32

# Test Results
Declan has tested it on his branch and it seems to work fine. Seems to have plenty of merge conflicts so, should be a fun one trying to merge this no Friday xD

### CODE REVIEW

File: files/web/backend/resources/Program_Manager.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+        elif(cmd == 'continue_program'):
+            # continue the supplied program (provided program is paused)
+            program_command = "sudo hobby-hub -c " + args["program"]
+            os.system(program_command)
+
+        elif(cmd == 'stop_program'):
+            # stop the supplied program completely (provided program is running)
+            program_command = "sudo hobby-hub -s " + args["program"]
+            os.system("sudo hobby-hub -s " + args["program"])
```
Comment (Ivendrakeo) [inline files/web/backend/resources/Program_Manager.py]: why is this one different? are you not using program_command?

**Comment by Ivendrakeo** [comment]
overall. looks pretty good. (assuming it works on the front end)

File: files/web/backend/resources/Program_Manager.py (hunk 1 above)
Comment (anthony-bartman) [reply files/web/backend/resources/Program_Manager.py]: I put it all into one class called program manager @Ivendrakeo

Comment (anthony-bartman) [reply files/web/backend/resources/Program_Manager.py]: Declan had get and post split into 2, so I just put into 1 for simplicity sake

**Pull request merged**